Backend/scala-backend/app/models/StudentModel.scala

```diff
@@ -70,7 +69,10 @@ class StudentModel(db: Database)(implicit ec: ExecutionContext) {
   }
 
   def updateGroupBelonging(studentId: Int, group: Int): Future[Int] = {
+    println(s"id searched: ${studentId}")
+    db.run(Student.filter(_.id === studentId).result).map(rows => println(s"students found in update: ${rows}"))
```

Comment:
manchmal ist dies einfach ein leerer vektor, obwohl es safe einen studenten mit der id in der datenbank gibt. Vorher wird nämlich getAllSelfreportedStudents aufgerufen, was erfolgreich die beiden ids zurückgibt